mwm-3-kotlin-1/presentation/PITCHME.md

```diff
@@ -546,6 +546,25 @@ public final void testForWithIndices() {
 
 ---
 
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Extend</span> <span style="text-transform: none; font-size:0.8em;">from a parent</span>
+
+In Kotlin, class and methods are `final` by default.
+
+Use `open` keyword in from of a class/method to be able to extend from it :
```

Comment:
Smaller `open keyword required to be able to extend from class/method:`

PS: "in from of" -> "in front of"

Reply:
Fixed !